app/mesh/1.4.x/features/vault.md

```diff
@@ -196,6 +198,7 @@ spec:
             namespace: "" # optional
             pki: kmesh-pki-default # name of the configured PKI
             role: dataplane-proxies # name of the role that will be used to generate data plane proxy certificates
+            commonName: '{{ tag "kuma.io/service" }}.mesh' # optional. If set, then common name is added to the certificate. You can use "tag" directive to pick a tag which will be base for common name.
```

Comment:
I'm not sure if this should match as it does in the previous comment, but if it is referring to the same thing consistency helps to avoid confusion

```suggestion
            commonName: '{{ tag "kuma.io/service" }}.mesh' # optional. If set, then commonName is added to the certificate. You can use "tag" directive to pick a tag which will be base for commonName.
```